### ISSUE TITLE
Test failure in opm-core

### ISSUE THREAD

**Issue by joakim-hove** [OPM/opm-core#894]
```
unknown location(0): fatal error in "DeckWithLiveOil": std::invalid_argument: TableContainer does not have any table in the range 0...0
/private/jenkins/jenkins/RH6/workspace/opm-core-install-RH6/tests/test_equil.cpp(518): last checkpoint

*** 1 failure detected in test suite "UnitsTest"
```

I'm on it.

**Comment by joakim-hove** [comment]
OK - fixed here: https://github.com/OPM/opm-core/pull/895

**Comment by brenoeduardots** [comment]
Hello joakim-hove. I have been finding the same error. I have been trying to test opm in a simple Eclipse project of dual phase oil and water with 2 saturations tables and 2 pvt tables. How did you solve it? Thanks in advance.

**Comment by joakim-hove** [comment]
Hello;

the error I reported was fixed more than 1.5 years ago - so I doubt we have the same problem, although the symptoms can of course be the same.

Please open a new Issue under opm-simulators with a detailed description of:

1. What you tried to do.
2. What problems you encountered.
3. *All* relevant error messages and backtraces.
